manifest.yaml

```
@@ -2,25 +2,20 @@
   name: AiiDA
   description: AiiDA environment for exploring AiiDA databases.
   variables:
-    description: Describe your project
     archive_url: (optional) URL to AiiDA archive to be imported upon starting an environment
-- folder: python-minimal-streamlit 
+- folder: python-minimal-streamlit
   name: Streamlit
   description: Streamlit template based on a basic Python (3.7) project.
-  variables:
-    description: Describe your project
+  variables: {}
 - folder: desktop
   name: Renku Desktop
   description: A Renku Desktop session with Python
-  variables:
-    description: Describe your project
+  variables: {}
 - folder: qgis
   name: QGIS application
   description: A popular, open-source geographic information system.
-  variables:
-    description: Describe your QGIS project
+  variables: {}
 - folder: matlab
   name: MATLAB
   description: A MATLAB client through Jupyter proxy
-  variables:
-    description: Describe your MATLAB project
\ No newline at end of file
+  variables: {}
```

```suggestion
  variables: {}

```
Just a missing new line at the end of this. All looks good otherwise